crud-archetype/src/main/resources/archetype-resources/__iospath__/pom.xml

```diff
@@ -40,7 +40,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>2.3.2</version>
```

Comment:
One thing I was considering is to engage http://frgaal.org - once frgaal compiler works, it basically untangles the Java language syntax away from the JDK and removes one pain point we always have to work with - e.g. Javac changing its behaviour.